manifest/sourcefile.py

```diff
@@ -274,11 +275,18 @@ def timeout_nodes(self):
 
     @cached_property
     def script_metadata(self):
-        if not self.name_is_worker and not self.name_is_multi_global:
+        if not (self.name_is_worker or
+                self.name_is_multi_global or
+                self.name_is_webdriver):
             return None
 
+        if self.ext == ".js":
+            regexp = js_meta_re
+        elif self.ext == ".py":
+            regexp = python_meta_re
```

Comment:
This would be more obviously correct as
```py
if self.name_is_worker or self.name_is_multi_global:
    regexp = js_meta_re
elif self.name_is_webdriver:
    regexp = python_meta_re
else:
    return None
```